src/Engine/Options.cpp

```diff
@@ -660,6 +660,8 @@ void refreshMods()
 	}
 	Log(LOG_INFO) << "Scanning user mods in '" << getUserFolder() << "'...";
 	FileMap::scanModDir(getUserFolder(), "mods", false);
+	Log(LOG_INFO) << "Scanning system mods in '" << getDataFolder() << "'...";
+	FileMap::scanModDir(getDataFolder(), "mods", false);
```

Comment:
This exact functionality is few lines below on line #672.
I have explicitly added it only for Android, and not other platforms.
Also, your change doesn't work, because it allows to load mods twice if the data folder and user folder are the same, which leads to issues.

Reply:
As I checked double loading make no issues. OXCE uses mod from UserFolder if two copies are available.

Reply:
It does create a lot of issues, I made the same mistake when I added the Android version, and had to release a hotfix for it.